### DISCUSSION TITLE
Troubles downloading installers ?

### DISCUSSION THREAD

**Discussion by SbiCA** [dotnet/aspnetcore#58325] [Q&A]
I'm having troubles downloading the installers.

Earlier today I couldn't download the RC2 installers the always returned 0-bytes.

Now I'm trying to install the latest 8 ARM64 for mac with the same result is there anyone else facing this issue?

https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/sdk-8.0.403-macos-arm64-installer

![image](https://github.com/user-attachments/assets/c9b47f7a-6f38-495c-bfe5-9454d9fa01ca)

**Comment by xC0dex** [comment]
There are couple of open issues in the different repos. One workaround that worked for me is to open the .NET download page in inkognito mode.
